resources/schema.gql

```diff
@@ -1827,7 +1850,7 @@ type TasksMut {
 	Schedules a task to update the specified dataset by performing polling
 	ingest or a derivative transformation
 	"""
-	createUpdateDatasetTask(datasetId: DatasetID!): Task!
+	createUpdateDatasetTask(datasetId: DatasetID!, fetchUncacheable: Boolean): Task!
```

Comment:
This wasn't worth it, I think we might remove this API entirely. UI will not allow direct tasks management anyway.